documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -96,6 +96,31 @@ Some models of Raspberry Pi, including Zero, Compute Module, and Keyboard models
 
 The `[none]` filter prevents any settings that follow from being applied to any hardware. Although there is nothing that you can't do without `[none]`, it can be a useful way to keep groups of unused settings in config.txt without having to comment out every line.
 
+=== The `[partition=N]` filter
+This `partition` filter can be be used to select alternate boot flows according to the requested partition number (`sudo reboot N`) or via direct usage of the `PM_RSTS` watchdog register.
+
+[source,ini]
+----
+# Bootloader EEPROM config - disable SD high speed and show HDMI diagnostics
+# Boot from partition 2 with debug option. Only high (>31) partition can be remapped.
+[partition=62]
+PARTITION=2
+SD_QUIRKS=0x1
+HDMI_DELAY=0
```

Comment:
Do these `SD_QUIRKS` and `HDMI_DELAY` settings also only apply when you've done `sudo reboot 62` ?
Or does the `[partition=62]` in the EEPROM config work differently to the `[partition=62]` in `/boot/firmware/config.txt` ?

Reply:
Values in conditional sections are only applied if the conditional is true. The conditional may be used in either config.txt or EEPROM config on Pi5. On Pi4 it just applies to the EEPROM config.

Reply:
Perhaps in this example it's worth adding the `[all]` filter at the end, just to demonstrate the filter being reset / cleared? :shrug: